### PR TITLE
adding-badges-section-in-newcomers-members-page

### LINKED ISSUE #4892: Add Mention of Badges Site to Layer5.io Pages
### Current Behavior

Layer5 recently announced an enhancement to the ways in which it recognizes milestones achieved by contributors and users. (see community handbook) with an improved badge award system --> https://badges.layer5.io/.
### Desired Situation

A short and crisp description of the badges site need to be added on the following pages

- [ ]  [community/newcomers](https://layer5.io/community/newcomers)

- [ ]  [community/members](https://layer5.io/community/members)

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by PRANJALRANA11**
**Description**
I have added in both pages
image

and i have updated the image align
image

This PR fixes #4892 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by PRANJALRANA11** [comment]
@leecalcote I have added the necessary changes

**Comment by l5io** [comment]
🚀 Preview for commit 5210a1a69f6c6418fb52d436d2ee1de743446706 at: https://64fd548261818b681c9fda26--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
hey @PRANJALRANA11 

Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)
https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Review by ShravaniAK — CHANGES_REQUESTED** [review]
@PRANJALRANA11 I would be better if the text and heading aligns , and also there is alignment issue in images . Also currently it is not mobile responsive .

**Comment by PRANJALRANA11** [comment]
@ShravaniAK ok sure I will do it

**Comment by l5io** [comment]
🚀 Preview for commit 14f027b0411791f79d1fc6cd9ac24a18f8fea79b at: https://65012e3dceeaf62b3fe0bfb6--layer5.netlify.app

**Comment by PRANJALRANA11** [comment]
@ritiksaxena124  please review the changes

**Comment by PRANJALRANA11** [comment]
@ShravaniAK please review the changes

File: src/sections/Community/Members-grid/membersGrid.style.js [hunk 1]
```diff
@@ -196,7 +196,11 @@ export const MembersGridWrapper = styled.div`
       padding-right: 3rem;
       padding-left: 3rem;
       margin-right: 3rem;
-      margin-top: 3rem;
+      margin-top: 2rem;
+    }
+    #Badge_img {
+      width: 444px;
```
Comment (Chadha93) [inline src/sections/Community/Members-grid/membersGrid.style.js:202]: @PRANJALRANA11 Please use rem

Comment (PRANJALRANA11) [reply src/sections/Community/Members-grid/membersGrid.style.js]: @Chadha93 resolved it

**Comment by PRANJALRANA11** [comment]
@Chadha93 i have fix it

**Review by leecalcote — CHANGES_REQUESTED** [review]
Please shrink the image size at each breakpoint.
Please keep the image with the text, wrapping the text around the image in mobile view.

Screenshot 2023-09-15 at 12 08 39 PM

**Comment by PRANJALRANA11** [comment]
@leecalcote fixed that in mobile view

**Comment by l5io** [comment]
🚀 Preview for commit 44b0b36fc930927a15f76df740f261d50f2f9a83 at: https://6505cd81a20b4e2a3a8ee5f6--layer5.netlify.app

**Review by abhijeetgauravm — APPROVED** [review]
Looks good!

**Comment by PRANJALRANA11** [comment]
@leecalcote is this good to merge

File: src/sections/Community/Members-grid/index.js [hunk 2]
```diff
@@ -108,6 +111,64 @@ const MembersGrid = (props) => {
...
+              open source projects. As a contributor, this is how you demonstrate your ownership,
+              dedication, skills and loyalty to Layer5. As a user,
+              this is how you demonstrate your prowess.
+              </p>
+              <p>
+                Whether as a user or a contributor,
+                community members are awarded badges as and when they
+                achieve any number of different milestones throughout
+                their journey whether that accomplishment lie within a
+                community activity or an aspect of using or contributing to a project .
```
Comment (leecalcote) [inline src/sections/Community/Members-grid/index.js:135]:
```suggestion
                community activity or an aspect of using or contributing to a project.
```

File: src/sections/Community/Members-grid/membersGrid.style.js [hunk 3]
```diff
@@ -159,11 +158,20 @@ export const MembersGridWrapper = styled.div`
   .members-profile-cards {
     z-index: -1 !important;
   }
+  #Badge_img {
```
Comment (leecalcote) [inline src/sections/Community/Members-grid/membersGrid.style.js:161]: We don't use ids (#). Please follow suit with the rest of the pre-established styling convention.

File: src/sections/Community/Newcomers-guide/index.js [hunk 4]
```diff
@@ -216,6 +217,32 @@ const NewcomersGuide = () => {
               </p>
             </Col>
           </Row>
+          <Row>
+            <Col sm={12} lg={6} style={{marginTop:"3rem"}}>
+              <h3>Recognizing Community Members</h3>
+              <ol type="i">
+                <li>
+                Whether as a user or a contributor, community members are awarded badges as and when they achieve any number of different milestones throughout their journey whether that accomplishment lie within a community activity or an aspect of using or contributing to a project
+                  .
```
Comment (leecalcote) [inline src/sections/Community/Newcomers-guide/index.js:226]:
```suggestion
                Whether as a user or a contributor, community members are awarded badges as and when they achieve any number of different milestones throughout their journey whether that accomplishment lie within a community activity or an aspect of using or contributing to a project.
```

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
> @leecalcote is this good to merge

Unfortunately, no. This is taking more reviews than it should.

File: src/sections/Community/Members-grid/membersGrid.style.js (hunk 3 above)
Comment (leecalcote) [reply src/sections/Community/Members-grid/membersGrid.style.js:161]: @PRANJALRANA11 did you incorporate this feedback?

**Comment by l5io** [comment]
🚀 Preview for commit d0641ca6ae423191b6626e5f024a7d9a543eac25 at: https://651c781e45250a1b317d5d31--layer5.netlify.app

File: src/sections/Community/Members-grid/membersGrid.style.js (hunk 3 above)
Comment (PRANJALRANA11) [reply src/sections/Community/Members-grid/membersGrid.style.js:161]: I will commit it by today

**Comment by PRANJALRANA11** [comment]
it's failing the install and build job

**Comment by leecalcote** [comment]
@PRANJALRANA11 that means that a bug has crept in. I'll let you fix.

**Comment by l5io** [comment]
🚀 Preview for commit 7c514dfd4940c001dd9e8a0adc77ed773ad114a2 at: https://651cf2d48306007d8360c41f--layer5.netlify.app

**Comment by PRANJALRANA11** [comment]
What was the bug

**Comment by l5io** [comment]
🚀 Preview for commit 5c76bea134a193964ba0abc23c374466da4c995a at: https://65488781f1655163cd2623c4--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@PRANJALRANA11 are you still there? at newcomers page the image getting pressed
image

**Pull request closed without merging**